numpy/random/mtrand.pyx

```diff
@@ -1547,7 +1547,7 @@ cdef class RandomState:
         >>> mu, sigma = 0, 0.1 # mean and standard deviation
         >>> s = np.random.normal(mu, sigma, 1000)
 
-        Verify the mean and the variance:
+        Verify the mean and the standard deviation:
```

Comment:
The same text is used in the new `Generator` API as well.  So if we change it here, should change it there also.

Reply:
Yes. Thanks for pointing that out. I have commited that now.